docs/src/index.md

```diff
@@ -126,7 +127,7 @@ r.converged             # whether the procedure converged
 - [Hamerly()](https://www.researchgate.net/publication/220906984_Making_k-means_Even_Faster) - Hamerly is good for moderate number of clusters (< 50?) and moderate dimensions (<100?).
 - [Elkan()](https://www.aaai.org/Papers/ICML/2003/ICML03-022.pdf) - Recommended for high dimensional data.
 - [Yinyang()](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/ding15.pdf) - Recommended for large dimensions and/or large number of clusters.
-- [Coreset()](http://proceedings.mlr.press/v51/lucic16-supp.pdf) - Recommended for very fast clustering of very large datasets, when extreme accuracy is not important.
+- [Coreset()](http://proceedings.mlr.press/v51/lucic16-supp.pdf) - Recommended for very fast clustering of very large datasets, when extreme accuracy is not important. *Experimental Implementation*
```

Comment:
Why is it experimental? As far as I remember it was complete, tested and working.

Reply:
What about the slow convergence mechanism? That needs to be fixed?

Reply:
Slow convergence? I guess I do not remember most of the package :-) As far as I remember, Coresets were the fastest algorithm possible. They have less accuracy, that's true, but this "less" in a sense, that it's hard to achieve ~10^-7 relative tolerance, which is way more accurate than needed in real-life applications. For reasonable tolerance, it is extremely fast.

Reply:
That comment was for mini-batch. fixed now :)